### PR TITLE
USWDS-Site - Icon: Add note about icon color contrast

### LINKED ISSUE #2224: USWDS-Site - Feature: Add WCAG Guidance on Color Contrast for Icon Component
### Is your feature request related to a problem? Please describe.

The existing accessibility documentation for the icon component does not provide guidance for color contrast as outlined by WCAG https://www.w3.org/WAI/WCAG21/Techniques/general/G207

### Describe the solution you'd like

Add guidance for ensuring icons have sufficient 3:1 color contrast against their background. Draft verbiage:

"**Check for sufficient color contrast.** Verify that there is at minimum a 3:1 color contrast of the icon against the background. 

Use [color contrast checking tools recommended by digital.gov](https://accessibility.digital.gov/visual-design/color-and-contrast/)"

### Describe alternatives you've considered

_No response_

### Additional context

More details on testing and procedure, along with examples, can be found on the [WCAG website](https://www.w3.org/WAI/WCAG21/Techniques/general/G207).

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate feature requests.

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary
Added a note about checking icon color contrast.

> **Warning**
> We should update the changelog date before merging

## Related issue

Closes #2224

## Preview link

[Icon component page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-icon-contrast-note/components/icon/#accessibility-icon)

## Testing and review

Confirm that the copy makes sense, conveys what we want it to, and is free from spelling and grammatical error.

### CODE REVIEW

**Comment by amyleadem** [comment]
@amycole501 Good point! I updated the link text to reflect the title of the article. Let me know if you want any changes!

**Review by mahoneycm — APPROVED** [review]
Lgtm! 👍

**Comment by mejiaj** [comment]
@sarah-sch would you be able to review & approve? We can merge after.

**Review by mejiaj — APPROVED** [review]

File: _components/icon/guidance/accessibility.md
Comment (sarah-sch) [inline _components/icon/guidance/accessibility.md:1]: **Check for sufficient color contrast.** Verify that the icon has a minimum contrast ratio of 3:1 against its background. See [WCAG 2.1 Techniques: Ensuring that a contrast ratio of 3:1 is provided 

@amyleadem - looked at this with @finekatie and we are suggesting some tweaks to maintain USWDS's conversational, approachable style: **Check for good color contrast.** Make sure that the icon has a minimum contrast ratio of 3:1 against its background.

**Review by sarah-sch — CHANGES_REQUESTED** [review]
Thanks for sharing, @amyleadem! Katie and I are suggesting some tweaks.

File: _components/icon/guidance/accessibility.md
Comment (amyleadem) [reply _components/icon/guidance/accessibility.md:1]: @sarah-sch @finekatie 
Good improvement! I've updated the copy in [c634d58](https://github.com/uswds/uswds-site/pull/2258/commits/c634d58b6d8891247014c34784a25f1e6b3de988). Let me know if you want any changes.

**Review by sarah-sch — APPROVED** [review]
Looks good--thanks!

**Pull request merged**